File: Iniciante/URI 1078.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var n = parseInt(lines.shift());
console.log('1 x ' + n + ' = ' + (n * 1));
console.log('2 x ' + n + ' = ' + (n * 2));
console.log('3 x ' + n + ' = ' + (n * 3));
console.log('4 x ' + n + ' = ' + (n * 4));
console.log('5 x ' + n + ' = ' + (n * 5));
console.log('6 x ' + n + ' = ' + (n * 6));
console.log('7 x ' + n + ' = ' + (n * 7));
console.log('8 x ' + n + ' = ' + (n * 8));
console.log('9 x ' + n + ' = ' + (n * 9));
console.log('10 x ' + n + ' = ' + (n * 10));

File: Iniciante/URI 1078.r
input = file('stdin', 'r')

n = as.integer(readLines(input, n=1))

write(sprintf("1 x %d = %d", n, n * 1), '')
write(sprintf("2 x %d = %d", n, n * 2), '')
write(sprintf("3 x %d = %d", n, n * 3), '')
write(sprintf("4 x %d = %d", n, n * 4), '')
write(sprintf("5 x %d = %d", n, n * 5), '')
write(sprintf("6 x %d = %d", n, n * 6), '')
write(sprintf("7 x %d = %d", n, n * 7), '')
write(sprintf("8 x %d = %d", n, n * 8), '')
write(sprintf("9 x %d = %d", n, n * 9), '')
write(sprintf("10 x %d = %d", n, n * 10), '')